### ISSUE TITLE
Tilix symbol lookup error in Kali

### ISSUE THREAD

**Issue by secdoc** [gnunn1/tilix#1761]
The following error/issue began yesterday after performing apt update&&apt upgrade and then later restarting system.

Error: `_symbol lookup error_ tilix_ undefined symbol_ _D4glib1c5types__T6ScopedTC3gio22ApplicationCommandLineQyZQBo6__dtorMFZv`

I have removed and re-installed and everything is installing fine...just not running and generating error when trying to start from terminal, otherwise there is no response. I have seen that there have been similar issues with others on various distros, but I have been running Tilix for a while and never had this occur before... Any ideas?

**Comment by rootleet** [comment]
So, it appears you have the same version I had which means you might be updating from the Kali servers. Removing Tilix and installing the latest stable v1.9.3 fixed this issue for me.

Remove Tilix
sudo apt remove tilix

From the Tilix website:

For 64-bit distros where a package is not available, Tilix can be installed manually from the Tilix Github releases section by downloading tilix.zip and following these instructions:

wget https://github.com/gnunn1/tilix/releases/download/1.9.3/tilix.zip
sudo unzip tilix.zip -d /
sudo glib-compile-schemas /usr/share/glib-2.0/schemas/

**Comment by avimehenwal** [comment]
**As of 4th-October-2019**, I had the same issue on when I installed tilix from apt for parrot OS

```
Distributor ID: Parrot
Description:    Parrot GNU/Linux 4.7
Release:        4.7
Codename:       n/a
```
Downloading the release from tilix website worked for me :)

Super thank for answer @rootleet

**Comment by rootleet** [comment]
@avimehenwal you welcome and did it work?

**Comment by avimehenwal** [comment]
Yes it did work for me .... @rootleet Thankyou for your solution

Lets the linux system fear me now  ...

**Comment by patonwhite** [comment]
@rootleet thanks heaps just worked for me!

**Comment by V-Saint** [comment]
_Thanks @rootleet for reposting my solution as your own._

### Kali is they are normally a few versions behind on most tool repositories.  When they update this package the issue will probably be remediated.

_**Until then... Removal and reinstall from directly Git worked for me.**_

**Check Tilix Version** (This was happening with v1.8.9-1 for me, but possibly in other versions.)
sudo apt show tilix
> Package: tilix
Version: 1.8.9-1
Priority: optional

**Remove Tilix**
sudo apt remove tilix

**Install Tilix**
wget https://github.com/gnunn1/tilix/releases/download/1.9.3/tilix.zip
sudo unzip tilix.zip -d /
sudo glib-compile-schemas /usr/share/glib-2.0/schemas/

I hope this helps other Kali Tilix users.

**Comment by NilangaPerera** [comment]
@rootleet Thank you 
You save my day. if any one who trying to get Tilix to work. 
Follow the steps below as @rootleet mentioned in his post. 

Remove Tilix
sudo apt remove tilix

wget https://github.com/gnunn1/tilix/releases/download/1.9.3/tilix.zip
sudo unzip tilix.zip -d /
sudo glib-compile-schemas /usr/share/glib-2.0/schemas/
